src/name_bazaar/server/contracts_api/registrar.cljs

```diff
@@ -31,7 +31,7 @@
                            opts))})
 
 (defn registration-owner [{:keys [:ens.record/hash :ens.record/label]}]
-  (contract-call :name-bazaar-registrar :ownerOf (sha3 label hash)))
+  (contract-call :name-bazaar-registrar :owner-of (sha3 label hash)))
```

Comment:
Also, I checked deep down in the district smart contracts dependency and both camelCase version and kebab-case should work. But we use kebab case for consistency.